src/event/share-event.js

```diff
@@ -1,15 +1,21 @@
 /**
  * Share plugin event types.
  */
-const ShareEvent = {
+export const ShareEvent = {
   /**
-   * Fired when the user clicks on share icon.
+   * Fired when the user clicks on share plugin icon.
    */
-  SHARE_CLICKED: 'share_clicked',
+  SHARE_EMBED_OPEN: 'Share_embed_open',
   /**
-   * Fired when the user clicks on specific social network.
+   * Fired when the user clicks on close (overlay) button.
    */
-  SHARE_NETWORK: 'share_network'
+  SHARE_EMBED_CLOSE: 'Share_embed_close',
+  /**
+   * Fired when the user clicks on any social network icon.
+   */
+  SHARE_EMBED_SHARE_VIA_SOCIAL_NETWORK: 'Share_embed_share_via_social_network',
```

Comment:
maybe add a namespace (either share of kaltura-player)
like here
https://github.com/kaltura/kaltura-player-js/blob/515fc71aa577ccc64c481ec98105ff78c31df615/src/common/cast/cast-event-type.ts#L25

Reply:
GA, will do

Reply:
Change of names is breaking change, if they are already there you cannot know if someone is using them

Reply:
The product requirement is that the name will be exactly as provided in the ticket 
These events were added for internal usage (insights)

Reply:
@JonathanTGold my recommendation is to not ignore this comment, events are public and if you cancel exiting ones you can crash a website

Reply:
Got it
Ok  I will either leave the previous ones for backward compatibility or consult the product to stick the current names here…